src/css/styles.css

```diff
@@ -158,19 +158,25 @@ body {
   opacity: 0.8;
 }
 
-.list-comment {
```

Comment:
Consider removing the font import and usage in line 1 and from lines 14-19, as we are not making use of this code.

Reply:
. list-comment is in use in the pop up modal

![image](https://user-images.githubusercontent.com/68148157/143500212-9650550c-89aa-451a-9008-cd71ac52e7bc.png)

Reply:
I placed the comment here as it didn't let me comment above. That is why I specified that my request was for line 1 and lines 14-19
![image](https://user-images.githubusercontent.com/59420634/143500414-1b396fbf-28b0-45e0-a0a4-19d2969ba37b.png)

Reply:
Highlighted removed